_assets/js/pagination.js

```diff
@@ -25,12 +26,12 @@ function reqLoaded() {
     let resJSON = JSON.parse(this.responseText);
     renderSearchPage(resJSON, urlParams, NUMBER_OF_RESULTS);
   } else {
-    renderSearchResults(`${noResults()}`, false);
+    target.innerHTML = totalResults(null, 'result');
   }
 }
 // If timeout:
 function reqTimeout() {
-  renderSearchResults(`${noResults()}`, false);
+  target.innerHTML = totalResults(null, 'result');
```

Comment:
Should this be null or 0?

Reply:
I have totalResults set to handle 0 differently than null. Null is passed in when there is an server side error. Here if the request timesout, we pass in null to return the error text. If 0 is passed in, then that means there are no results, so we return the no results found text.

Reply:
I usually try and keep each variable as one data type.  I wonder if you could pass in -1 instead?  If you want to use null though, its cool.

Reply:
That makes sense. I could pass in -1, I just need to alter the if checks because one is just checking if the number is less than 1. So I'll add one that checks explicitly for 0, and another that explicitly checks for -1.